src/test/java/api/loans/anonymization/AnonymizeLoansAfterXIntervalTests.java

```diff
@@ -411,7 +419,10 @@ public void testClosedLoanWithOpenFeesAndFinesNotAnonymizedIntervalPassed() {
 
 
     createOpenAccountWithFeeFines(loanResource);
-    DateTimeUtils.setCurrentMillisOffset(ONE_MINUTE_AND_ONE);
+
+    mockClockManagerToReturnFixedDateTime(
+      DateTime.now(DateTimeZone.UTC).plus(ONE_MINUTE_AND_ONE));
```

Comment:
Personally, I might statically import `now` and `UTC` if it might make this more readable and fit on a single line

Reply:
Sure.